app/views/state_file/questions/unemployment/index.html.erb

```diff
@@ -12,7 +12,7 @@
             <%= t('.1099_label', name: form1099.recipient_name) %>
           </h3>
           <p class="form1099__info spacing-below-25">
-            <%= t('.unemployment_compensation', amount: form1099.unemployment_compensation_amount, precision: 2) %>
+            <%= t('.unemployment_compensation', amount: number_to_currency(form1099.unemployment_compensation_amount, precision: 2)) %>
```

Comment:
this was a bug -- we need to pass the `precision` arg to `number_to_currency` -- otherwise `t` method was thinking the `precision` was another variable passed to the text.